.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
@rusefillc ChibiOS `stable_20.3.x.rusefi` uses CRLF line-endings -- I made my changes against `master` which had LF line-endings ... this addition (and `git add --renormalize os/hal/ports/STM32` made fixing my rebase easier -- is this okay or should we avoid this, or perhaps renormalize the entire tree?

Reply:
why is your git line ending mode set to the mode where it doesn't change to your machine's culture?

Reply:
It is setup with auto/native, but for some reason `rebase` wasn't having it with the difference and generating full-file 3-ways.  I'm happy to remove this and retain CRLF in the repository, but I believe that's the culprit (the committed files are CRLF, and `rebase` wasn't detecting them as text).

Reply:
Oh shit, `core.autocrlf` was in fact commented out in my config. :+1: